apps/electron/package.json

```diff
@@ -68,16 +68,6 @@
     "rxjs": "^7.8.1",
     "yjs": "^13.6.7"
   },
-  "build": {
-    "protocols": [
-      {
-        "name": "affine",
-        "schemes": [
-          "affine"
-        ]
-      }
-    ]
-  },
```

Comment:
It may be useful for mac. I think the tool which you choose to build electron will write `CFBundleURLSchemes` in Info.plist

Reply:
This has been moved to https://github.com/toeverything/AFFiNE/pull/3579/files/e29dc013ea312836fbb6b3278f39a32e8c3ade1a#diff-55bbe6084a272e4351107e32b1689f3a9a351296fc2730a959122b2861d0199eR133-R138